### PR TITLE
Fixed deprecation warnings PHP8+ in helper/handler.

### PR DESCRIPTION
**Pull request by nerun**
This PR fix deprecation warnings in PHP 8+ and Jack Jackrum: PageResolver and MediaResolver.

Related to issue #241

### CODE REVIEW

**Comment by nerun** [comment]
I don't know why your tests have failed, but it's working here: page move, page rename etc.

File: helper/handler.php [hunk 1]
```diff
@@ -66,7 +69,7 @@ public function resolveMoves($old, $type) {
 
         if($type != 'media' && $type != 'page') throw new Exception('Not a valid type');
 
-        $old = resolve_id($this->origNS, $old, false);
+        $old = (new PageResolver($this->origNS))->resolveId($old);
```
Comment (michitux) [inline helper/handler.php:72]: `resolve_id` is not the same as `PageResolver`. I assume, but I'm not sure, that this breaks the tests as you can see from the checks. The test breakages look quite strange, as if there was a quite fundamental error somewhere. I don't know where it comes from but please run the tests and debug where the differences come from.

**Review by michitux — CHANGES_REQUESTED** [review]
As the tests show, this is not a simple refactoring but breaks the plugin. Please fix the code so the tests pass again.

**Comment by michitux** [comment]
> I don't know why your tests have failed, but it's working here: page move, page rename etc.

The failing tests concern the adaptation of the links after the move, not the actual moving of the page.

File: helper/handler.php (hunk 1 above)
Comment (nerun) [reply helper/handler.php]: Changed from PageResolver to FileResolver.

**Comment by nerun** [comment]
Not working. Don't know how to fix. Closing.

**Pull request closed without merging**